gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -77,9 +77,27 @@ to generate set of instructions that includes a command block with the registrat
 // tag::reg-command-intro[]
 To register a Flex Gateway with Anypoint Platform, you must enter the registration command and then the start command. Each command includes information specific to your Anypoint Platform instance and *must be updated before* executing. See <<Before You Begin>> for more information on how to find the information you will need.
 //end::reg-command-intro[]
+//
+// tag::k8s-connected-intro[]
+In this mode, the gateway is managed entirely through Anypoint Platform, where you can: 
+
+* Deploy APIs to your cluster.
+* Apply policies and other configurations.
+* Create all resources through Anypoint Platform. 
+
+Resource creation through `kubectl apply` is restricted to specific resource types.
+
+// end::k8s-connected-intro[]
+//
+// tag::k8s-local-intro[]
+In Local Mode, Flex Gateway acts as an ingress controller that manages external access to your Kubernetes cluster. When using this mode, you must know what resources to create and apply, and use `kubectl apply` to deploy resources such as APIs, policies, or Flex Gateway-related resources to your gateway. Local Mode also supports the Ingress resources from Kubernetes, which are configured through a YAML file with properties, such as `apiVersion`, `kind`, `metadata`, `spec` for configuring a load balancer or proxy server, and `rules` for directing HTTP and HTTPS traffic (see https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes documentation^] on Ingress resources).
```

Comment:
I suggest using the ingress controller use case as an example, but not say that it's the only one because there are other use cases covered by flex in local mode
```suggestion
In Local Mode, Flex Gateway can act as an ingress controller that manages external access to your Kubernetes cluster. When using this mode, you must know what resources to create and apply, and use `kubectl apply` to deploy resources such as APIs, policies, or Flex Gateway-related resources to your gateway. Local Mode also supports the Ingress resources from Kubernetes, which are configured through a YAML file with properties, such as `apiVersion`, `kind`, `metadata`, `spec` for configuring a load balancer or proxy server, and `rules` for directing HTTP and HTTPS traffic (see https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes documentation^] on Ingress resources).
```

Reply:
@seblaz what other ways can Flex act besides an ingress controller? Is ingress controller something we recommend in terms of Flex specifically? Do we care how Flex acts? What ways do we recommend, if any?

Reply:
Basically, I would like to be able to say something that provides a bit of guidance about the use case we thing customers are likely to have for Flex, like "Flex Gateway typically acts as an ingress controller" or, for example (just an example I am making up), that we recommend using it as an ingress controller when in local mode, or whatever guidance we can give.

Reply:
@seblaz What's the common use case, and do we/MuleSoft have any opinion about how people should use Flex in local mode? And do people usually use it as an ingress controller in this case vs. in connected mode. And how do they usually use it in connected mode? Thx!

Reply:
> what other ways can Flex act besides an ingress controller? 

Mainly, as an ingress controller or as a standalone gateway


> Is ingress controller something we recommend in terms of Flex specifically?

It's probably the most common use case for local mode, so I think it makes sense to mention it here


> What ways do we recommend, if any?

I don't think we can make a recommendation about the use cases, it depends on the context where the users want to use it


> Basically, I would like to be able to say something that provides a bit of guidance about the use case we thing customers are likely to have for Flex, like "Flex Gateway typically acts as an ingress controller" or, for example (just an example I am making up), that we recommend using it as an ingress controller when in local mode, or whatever guidance we can give.

"Flex Gateway can act as an ingress controller or standalone gateway" covers most cases, so I think that should be good enough

> And how do they usually use it in connected mode? 

Usually as an ingress (note that this is not the same as ingress controller), or as a standalone gateway as well